developers/weaviate/configuration/indexes.md

```diff
@@ -102,20 +102,23 @@ Note that the vector index type only specifies how the vectors of data objects a
 
 ## Inverted index
 
-### Configure the inverted index
+### Configure the inverted index - for `text`
```

Comment:
not sure "for `text`" fits here. Inverted index can be applied to almost all types of data

Reply:
Fixed! :)